fluent-dom/src/localization.js

```diff
@@ -82,12 +99,10 @@ export default class Localization {
   /**
    * Retrieve translations corresponding to the passed keys.
    *
-   * A generalized version of `DOMLocalization.formatValue`. Keys can
-   * either be simple string identifiers or `[id, args]` arrays.
```

Comment:
Except that they cannot be simple strings. We do not support it at the moment :)